devcycle_python_sdk/managers/config_manager.py

```diff
@@ -1,27 +1,72 @@
+import threading
+import time
 import logging
+import json
 
 from devcycle_python_sdk.dvc_options import DevCycleLocalOptions
 from devcycle_python_sdk.api.local_bucketing import LocalBucketing
+from devcycle_python_sdk.api.config_client import ConfigAPIClient
+from devcycle_python_sdk.exceptions import CloudClientUnauthorizedError, CloudClientError
 
 logger = logging.getLogger(__name__)
 
 
-class EnvironmentConfigManager:
+class EnvironmentConfigManager(threading.Thread):
     def __init__(self, sdk_key: str, options: DevCycleLocalOptions, local_bucketing: LocalBucketing):
-        self.sdk_key = sdk_key
-        self.options = options
-        self.local_bucketing = local_bucketing
+        super().__init__()
 
-        # initialize Config API client
-        # TODO
-        # setup polling thread
-        # add initialized callback
+        self._sdk_key = sdk_key
+        self._options = options
+        self._local_bucketing = local_bucketing
+
+        self._config: dict = None
```

Comment:
i'm wondering if we even need to store the parsed json version of the config. We don't use it in the SDK for anything and just end up passing an encoded string to the WASM

Reply:
this more exists to denote that the config has been set at least once